packages/preview/typst-railroad/0.1.0/examples/simple.typ

```diff
@@ -0,0 +1,21 @@
+#import "../lib.typ": *
```

Comment:
Ideally this should use the package specification, to make it easier to get started from the example.

Reply:
Is this a new requirement? Star imports are fairly idiomatic

Reply:
What I meant by "package specification" was to use `#import "@preview/typst-railroad:0.1.0": *` (but with the new package name once you will have decided on it). Sorry if it was not clear.

Reply:
I think @elegaanz  means it should be like `#import @preview/....`, I was testing it locally, and using it that way does not work, I will change it for clarity.